spec/features/course/assessment/assessment_attempt_spec.rb

```diff
@@ -117,6 +117,9 @@
             expect(page).to have_selector('.submission_answers_grade')
           end
         end
+
+        # Check that publisher is set
+        expect(page).to have_content(submission.publisher.name)
```

Comment:
This could be the user name at sidebar. Should be more specific:
`expect(page).to have_selector('some selector', text: submission.publisher.name)`

Or just
`expect(submission.reload.publisher).to eq(sth)`